src/EditorFeatures/CSharpTest/Outlining/RegionDirectiveOutlinerTests.cs

```diff
@@ -89,5 +89,21 @@ static void Main(string[] args)
 }
 ");
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Outlining)]
+        [WorkItem(4105, "https://github.com/dotnet/roslyn/issues/4105")]
+        public void SpacesBetweenPoundAndRegionShouldNotAffectBanner()
```

Comment:
It's not an issue in VB, because VB includes the whole `#Region` line in the collapsed section and the banner.